### PR TITLE
Remove Helm support from test framework

### PR DESCRIPTION
**Pull request by howardjohn**

### CODE REVIEW

File: tests/integration/security/webhook/webhook_test.go [hunk 1]
```diff
@@ -56,10 +56,7 @@ func TestWebhookManagement(t *testing.T) {
 	framework.
 		NewTest(t).
 		Run(func(ctx framework.TestContext) {
-			cfg := inst.Settings()
-			if cfg.IsIstiodEnabled() {
-				ctx.Skip("TODO(github.com/istio/istio/issues/20289)")
-			}
+			ctx.Skip("TODO(github.com/istio/istio/issues/20289)")
```
Comment (nmittler) [inline tests/integration/security/webhook/webhook_test.go]: do we lose the ability to check for istiod being enabled?

Comment (howardjohn) [reply tests/integration/security/webhook/webhook_test.go]: I suppose we do, but we also don't have any tests that actually have istiod disabled and I was planning to remove the optional to disable istiod anyways so I figured I would just remove it? wdyt?

Comment (nmittler) [reply tests/integration/security/webhook/webhook_test.go]: ok sgtm

**Review by nmittler — APPROVED** [review]

File: tests/integration/security/webhook/webhook_test.go (hunk 1 above)
Comment (yangminzhu) [reply tests/integration/security/webhook/webhook_test.go]: cc @lei-tang

**Review by yangminzhu — APPROVED** [review]

**Pull request merged**